pkgs/tools/security/rekor/default.nix

```diff
@@ -1,21 +1,25 @@
 { lib, buildGoModule, fetchFromGitHub }:
 
 let
-  generic = { pname, subPackages, description, postInstall }:
+  generic = { pname, packageToBuild, description }:
     buildGoModule rec {
       inherit pname;
-      version = "0.1.1";
+      version = "0.2.0";
 
       src = fetchFromGitHub {
         owner = "sigstore";
         repo = "rekor";
         rev = "v${version}";
-        sha256 = "1hvkfvc747g5r4h8vb1d8ikqxmlyxsycnlh78agmmjpxlasspmbk";
+        sha256 = "1y6qw55r30jgkcwc6434ly0v9dcfa2lc7z5djn7rjcqrjg3gn7yv";
       };
 
-      vendorSha256 = "0vdir9ia3hv27rkm6jnvhsfc3mxw36xfvwqnfd34rgzmzcfxlrbv";
+      vendorSha256 = "1wlh505ypwyr91wi80fpbap3far3fljwjd4mql2qcqgg0b1yay9s";
 
-      inherit subPackages postInstall;
+      subPackages = [ packageToBuild ];
+
+      preBuild = ''
+        buildFlagsArray+=("-ldflags" "-s -w -X github.com/sigstore/rekor/${packageToBuild}/app.gitVersion=v${version}")
+      '';
```

Comment:
Please use ldflags array.

Reply:
you want it moved back to this format??

```
buildFlagsArray = [
  "-ldflags="
  "-s"
  "-w"
  "-X github.com/sigstore/rekor/${packageToBuild}/app.gitVersion=v${version}"
];
```

Reply:
We have something new and fancy https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/go.section.md#ldflags-var-go-ldflags

Reply:
nice, I'll make sure to change my packages next time I update them :+1: 

Is there a way to find out about these kinds of changes other than word-of-mouth?

Reply:
> Is there a way to find out about these kinds of changes other than word-of-mouth?

I don't think so.

Reply:
Thanks for the info :+1: . I will take care of it next time.